<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recept</class>
 <widget class="QWidget" name="recept">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="name_recept">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
color: #00007f;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="list_ingr">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
border: 1px solid #00007f;
border-radius: 10px;
color: #00007f;
background-color: #fff;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="disc_recept">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
border: 1px solid #00007f;
border-radius: 10px;
color: #00007f;
background-color: #fff;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="quitBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton, QPushButton:focus, QPushButton:focus {
font-weight: bold;
border: 1px solid #00007f;
border-radius: 15px;
color: #00007f;
background-color: #fff;
}

QPushButton:hover, QPushButton:hover:focus {
font-weight: bold;
border: 1px solid #00007f;
border-radius: 15px;
color: #00007f;
background-color: #eaffff;
}

QPushButton:pressed,QPushButton:pressed:focus {
font-weight: bold;
border: 1px solid #00007f;
border-radius: 15px;
color: #fff;
background-color: #00007f;
}</string>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked(bool)</signal>
   <receiver>recept</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
